plugin/src/test/java/org/opensearch/ml/processor/MLInferenceSearchResponseProcessorTests.java

```diff
@@ -503,6 +505,81 @@ public void onFailure(Exception e) {
         verify(client, times(1)).execute(any(), any(), any());
     }
 
+    /**
+     * Tests the successful processing of a response with a single pair of input and output mappings.
+     * read the query text into model config
+     * with query extensions
+     * @throws Exception if an error occurs during the test
+     */
+    public void testProcessResponseSuccessWriteToExt() throws Exception {
```

Comment:
let's add `@Test` for every test methods.

Reply:
added